### ISSUE TITLE
How To Finetune The Arabic Text Detection ?

### ISSUE THREAD

**Issue by MohieEldinMuhammad** [PaddlePaddle/PaddleOCR#7473]
I'm sorry for my confusion but can I get a step by step guide on how to prepare an arabic dataset and train ppocrv3 on it, also how to evaluate the training accuracy ? I need to do this in python.

I'm also wondering of the arabic dataset that the multilingual model trained on it available ?

**Comment by mga369** [comment]
how sway
Sent from Yahoo Mail on Android 
 
  On Sat, Sep 3, 2022 at 5:06, ***@***.***> wrote:   


I'm sorry for my confusion but can I get a step by step guide on how to prepare an arabic dataset and train ppocrv3 on it, also how to evaluate the training accuracy ? I need to do this in python.

I'm also wondering of the arabic dataset that the multilingual model trained on it available ?

**Comment by MohieEldinMuhammad** [comment]
I don't get it !

**Comment by andyjpaddle** [comment]
first, you can prepare your dataset by using PPOCRLabel https://github.com/PaddlePaddle/PaddleOCR/tree/release/2.6/PPOCRLabel, or you can use the datasets https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/dataset/vertical_and_multilingual_datasets_en.md#multi-language-datasetmulti-lingual-scene-text-detection-and-recognition
then, you can finetune your model by using the released pre-trained model

**Comment by MohieEldinMuhammad** [comment]
Thanks for the answer, Can you give me one sample of the detection and recognition train set to imagine how i will collect the data ? and what the model is expecting as an input in both detection and recognition.

**Comment by andyjpaddle** [comment]
you can refer to this doc https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/dataset/ocr_datasets_en.md

**Comment by MohieEldinMuhammad** [comment]
So when i download this data for example ```ICDAR2017 Competition on Multi-lingual scene text detection and script identification```
for detection and recognition i will need to format a new .txt file with both the coordinates and text as mentioned in the link you send

**Comment by andyjpaddle** [comment]
yes, you are right, but it's easy to transfer the label to a standard .txt file

**Comment by MohieEldinMuhammad** [comment]
Thanks 🙏
